config/crd/bases/controlplane.cluster.x-k8s.io_rosacontrolplanes.yaml

```diff
@@ -45,6 +45,21 @@ spec:
             type: object
           spec:
             properties:
+              MachineType:
+                description: The compute machine type to use, for example `r5.xlarge`.
+                  Machine type ref; https://aws.amazon.com/ec2/instance-types/
+                type: string
+              autoscaling:
+                description: Autoscaling specifies auto scaling behaviour for this
+                  NodePool.
```

Comment:
I believe the general preference is to refer to `MachinePool` and not `NodePool`. There is no customer-visible  mention of `NodePool` within ROSA CLI.